apps/mobile/src/features/approver/stacks-fees.section.tsx

```diff
@@ -0,0 +1,76 @@
+import { useRef, useState } from 'react';
+
+import { useCalculateStacksTxFees } from '@/queries/stacks/fees/fees.hooks';
+import { deserializeTransaction } from '@stacks/transactions';
+
+import { FeeTypes } from '@leather.io/models';
+import { Approver, Box, SheetRef } from '@leather.io/ui/native';
+import { createMoney } from '@leather.io/utils';
+
+import { StacksFeeCard } from './components/fees/stacks-fee-card';
+import { StacksFeesSheet } from './components/fees/stacks-fee-sheet';
+import { TxOptions, formReviewTxSummary } from './utils';
+
+interface StacksFeesSectionProps {
+  txHex: string;
+  setTxHex(txHex: string): void;
+  txOptions: TxOptions;
+}
+
+export function StacksFeesSection({ txHex, setTxHex, txOptions }: StacksFeesSectionProps) {
+  const tx = deserializeTransaction(txHex);
+  const { feeMoney } = formReviewTxSummary({
+    tx,
+    symbol: 'STX',
+  });
+  const { data: stxFees } = useCalculateStacksTxFees(tx);
+  const fee = tx.auth.spendingCondition.fee;
+  function getFeeType() {
+    const estimates = stxFees?.estimates;
+    if (Number(fee) === estimates?.[1]?.fee.amount.toNumber()) {
+      return FeeTypes.Middle;
+    }
+    if (Number(fee) === estimates?.[0]?.fee.amount.toNumber()) {
+      return FeeTypes.Low;
+    }
+    if (Number(fee) === estimates?.[2]?.fee.amount.toNumber()) {
+      return FeeTypes.High;
+    }
+    return FeeTypes.Custom;
+  }
+
+  const feeSheetRef = useRef<SheetRef>(null);
+
+  const [selectedFeeType, setSelectedFeeType] = useState<FeeTypes>(getFeeType());
+  const fees = {
+    [FeeTypes.Low]: stxFees?.estimates[0]?.fee || createMoney(0, 'STX'),
```

Comment:
How about just calling `createMoney(0, 'STX')` once and setting it in a constant?